docs/styles/a11y-styles.scss

```diff
@@ -0,0 +1,29 @@
+// -----
+// Components
+// -----
+
+// Custom variables import
+// @import ...
+
+// Alpaca core-components
+@import 'alpaca-components';
+
+// Custom components import
+// @import ...
+
+@import 'tests/a11y-test';
+
+// -----
+// Documentation
+// -----
+
+@import 'docs-only-styles/checkbox';
+@import 'docs-only-styles/button';
+@import 'docs-only-styles/grid';
+@import 'docs-only-styles/icons';
+@import 'docs-only-styles/list';
+@import 'docs-only-styles/minicart-product';
+@import 'docs-only-styles/newsletter';
+@import 'docs-only-styles/product';
+@import 'docs-only-styles/range-filter';
+@import 'docs-only-styles/search';
```

Comment:
It's kinda pointless to load docs-only-styles, b/c it doesn't work with a11y preview enabled.
The workaround will be to add `docs-only-styles` to the `a11y` preview too.